### DISCUSSION TITLE
Is it possible to add HubDuino?

### DISCUSSION THREAD

**Discussion by markbellkosel** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#294] [Ideas]
I am trying to figure how to add another way to communicate with the module. I know it can be done but it is far outside my abilities. I am currently running Home Assistant solely for the purpose of being a go between with this.

This is what I am trying to add....

https://github.com/DanielOgorchock/ST_Anything/blob/master/HubDuino/README.md

**Comment by chunkysteveo** [comment]
MQTT is the most universal communication language between IoT devices, surely Hubitat allows for MQTT?

**Reply by chunkysteveo** [reply]
Huh, turns out it doesn't and you need another node, like Homeassistant running an MQTT broker like Mosquito.

**Reply by markbellkosel** [reply]
Correct. I have that currently in use. Then I have a Home Assistant integration in Hubitat.

What I am looking to do is make the module talk directly to Hubitat.
